src/installation/guides/zfs.md

```diff
@@ -0,0 +1,225 @@
+on it will be destroyed.
+
+For a BIOS/MBR system:
+
+```
+Partition    Size       Type
+1             ?G     Solaris Root(bf00)
+```
+
+For a BIOS/GPT system:
+
+```
+Partition    Size       Type
+1             2M     BIOS boot    
+2             ?G     Solaris Root
+```
+
+ZFS pools should use IDs to maintain universal functionality across computers
+and to prevent the misrecognition of drives.
+
+IDs can be located in the `/dev/disk/by-id` directory:
+
+Note: `wwn-` entries and `ata-` entries are referencing equivalent drives and
+either can be used.
+
+```
+$ ls -l /dev/disk/by-id
+total 0
+lrwxrwxrwx 1 root root 10 Apr 16 17:34 ata-Hitachi_HDS721010CLA332_JP6911HD2AHS9F-part1 -> ../../sda1
+lrwxrwxrwx 1 root root 10 Apr 16 17:34 wwn-0x5000cca373e0f5d9-part1 -> ../../sda1
+```
+
+Note: Many users with Advanced Format hard drives will want to also add the `-o
+ahisft=12` argument. A value of 12 will set the drive's logical sectors to 4k in
+size.
+
+Create a pool specifying the previously created Solaris Root Partition from the
+disk:
+
+`# zpool create -f -m none zroot dev`
+
+| Argument | Action                                            |
+|----------|---------------------------------------------------|
+| -f       | Force the creation of the pool                    |
+| -m none  | Set the mountpoint to none                        |
+| zroot    | The name of the pool                              |
+| dev      | The device to be used in the creation of the pool |
+|          | (ie. ata-XXXXX-partX or wwn-XXXX-partX)           |
+
+To ensure the pool's creation was successful use:
+
+```
+$ zpool list
+NAME    SIZE  ALLOC   FREE  CKPOINT  EXPANDSZ   FRAG    CAP  DEDUP    HEALTH  ALTROOT
+zroot   928G   432K   928G        -         -     0%     0%  1.00x    ONLINE  -
+```
+
+Then create the following data sets, which are ZFS' equivalent to partitions:
+
+```
+# zfs create -o mountpoint=none zroot/ROOT
+# zfs create -o mountpoint=/ zroot/ROOT/void
+cannot mount '/': directory is not empty
+property may be set but unable to remount filesystem
+```
+
+At this point users may want to set additional options to their datasets. An
+explanation of the available options is avaiable under the Native Properties
+section of [zfs(8)](https://man.voidlinux.org/zfs.8#Native_Properties).
+
+## Base Installation
+
+To mount the system for installation, export then re-import the pool at an
+alternate mountpoint:
+
+```
+# zpool export zroot
+# zpool import -R /mnt/void zroot
+```
+
+**Important:** Please consult the Base Installation & Configuration sections,
+ignoring the configuration of a fstab, of the [chroot guide](./). These provide
+an in depth guide of a general configuration that is required before continuing.
+
+After consulting the guide, install `grub` and `zfs` from the new system which
+will build the requirements necessary to boot:
+
+`(chroot)# xbps-install -S -R $REPO grub zfs`
```

Comment:
I'd prefer if you only mentioned the packages themselves, without the command. If you feel users need an example for how to install packages, make something like `[install](../../xbps/index.md)` (this should be a link to where package installation is explained).